cpp/mail_krimdok_newsletter.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -o errexit -o nounset
+
+
+cd .
+
+
+most_recent_email_contents=$(ls -Art | tail -n 1)
+if [ -z "$most_recent_email_contents" ]; then
+    echo "No newsletters found"
+    exit 0
+fi
+
+time_since_last_modification=$(($(date +%s) - $(stat --format=%Y -- "$most_recent_email_contents")))
+if [ $time_since_last_modification -gt 86400 ]; then
```

Comment:
This does not really look fail-safe, it might be better to rename the file to a ".sent" extension which is skipped when looking for new newsletters, or something like that.